### PR TITLE
Update WindowsBuild.md

### PR DESCRIPTION
**Pull request by compnerd**
Clean up some of the windows recommendations.  This is motivated by trying to setup a clean environment with @keith.

- migrate from `toolchain` to `llvm-project` for the LLVM project
- extract `swift` and `cmark` from the `llvm-project` directory structure
- use the `EXTERNAL_PROJECT` and `ENABLE_PROJECTS` properly
- convert artifact fetching to use a command line tool to fetch the URLs rather than manually using a browser

This now reflects the standard checkout layout more closely.

Replace this paragraph with a description of your changes and rationale. Provide links to external references/discussions if appropriate.

Resolves SR-NNNN.

### CODE REVIEW

**Comment by compnerd** [comment]
@swift-ci please smoke test

File: docs/WindowsBuild.md [hunk 1]
```diff
@@ -78,15 +78,16 @@ git clone https://github.com/compnerd/swift-build swift-build
...
 
-Go to [compnerd's swift-build azure page](https://dev.azure.com/compnerd/swift-build/_build) and open [Pipelines](https://dev.azure.com/compnerd/swift-build/_build) where you'll see bots (hopefully green) for:
-
-- [ICU](https://dev.azure.com/compnerd/swift-build/_build?definitionId=9)
-- [SQLite](https://dev.azure.com/compnerd/swift-build/_build?definitionId=12&_a=summary)
-- [curl](https://dev.azure.com/compnerd/swift-build/_build?definitionId=11&_a=summary)
-- [libxml2](https://dev.azure.com/compnerd/swift-build/_build?definitionId=10&_a=summary)
-- [zlib](https://dev.azure.com/compnerd/swift-build/_build?definitionId=16&_a=summary)
+```
+python -m pip install --user msrest azure-devops tabulate
```
Comment (keith) [inline docs/WindowsBuild.md:82]: FWIW `python` does nothing for me, I have to do `C:\Python27\python.exe

**Comment by keith** [comment]
Nice!

File: docs/WindowsBuild.md (hunk 1 above)
Comment (compnerd) [reply docs/WindowsBuild.md:82]: Ugh, okay, lets change that to `C:\Python27\python.exe` then; this is part of the problem - python needs to be by absolute path :-(.

**Pull request merged**